source/_posts/2018-01-08-how-to-quickly-deploy-a-vuejs-app-to-heroku.markdown

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "How to quickly deploy a VueJS app to Heroku"
+category: javascript
+comments: true
+author:
+  name: Jorge Bejar
+  email: jorge@wyeworks.com
+  twitter_handle: jmbejar
+  github_handle: jmbejar
+  image: /images/team/jorge-bejar.jpg
+  description: Love to explore and write about Ruby, Javascript, full-stack development and technology in general.
+---
+
+Recently, I was investing some time learning VueJS and I found that it is a very interesting framework to play with. In fact, I've been working on a new project prototype during a few days and wanted to show it to some people, so I wanted to publish it anywhere in the Internet.
+
+I decided to put the project on Heroku so I started to research what is the best way to do that. To my surprise, I didn't find much about it apart from a few posts like [this](https://codeburst.io/quick-n-clean-way-to-deploy-vue-webpack-apps-on-heroku-b522d3904bc8) and [this](https://medium.com/netscape/deploying-a-vue-js-2-x-app-to-heroku-in-5-steps-tutorial-a69845ace489). 
```

Comment:
put the project on Heroku -> deploy the project to Heroku ?

Reply:
to do that -> to do it ?

Reply:
![image](https://user-images.githubusercontent.com/14424870/34611677-3ac9f534-f205-11e7-95dc-e1c308d7c505.png)
I think it would be nice to have something like the posts names as the label for the `href` instead of "this", WDYT?